### ISSUE TITLE
Is there a way to control the padding of `xlabel` and `ylabel`?

### ISSUE THREAD

**Issue by frankwswang** [GiovineItalia/Gadfly.jl#1514]
![Snipaste_2021-01-18_17-16-12](https://user-images.githubusercontent.com/41162655/104967337-1d43d400-59b1-11eb-9ae3-f7cd6003622b.png)

The spacing between the label and x-axis seems a bit too large. Is there a function to handle this? I can find any in the documentation. Thank you!

**Comment by Mattriks** [comment]
Not yet, but see https://github.com/GiovineItalia/Gadfly.jl/issues/1458#issuecomment-762678958

**Comment by frankwswang** [comment]
Yeah. I agree that forming a new struct for all different kinds of text (such as `label`s, `key`s, `title`s) in the plot to control the fields is a nice suggestion!

**Comment by frankwswang** [comment]
> Not yet, but see [#1458 (comment)](https://github.com/GiovineItalia/Gadfly.jl/issues/1458#issuecomment-762678958)

Yeah. I agree that forming a new struct for all different kinds of texts (such as `label`, `key`, `title`) in the plot to control the fields is a nice suggestion!
